### PR TITLE
DOCS-1406/1407 - updated metrics section

### PR DESCRIPTION
**Pull request by cswatt**
### What does this PR do?

- added advanced filtering page
- updated metrics summary page
- updated metrics index with a link to the new advanced filtering page, as well as a link to metrics types

### Motivation
dreams

### Preview

https://docs-staging.datadoghq.com/cswatt/advanced-filtering/metrics/
https://docs-staging.datadoghq.com/cswatt/advanced-filtering/metrics/advanced-filtering
https://docs-staging.datadoghq.com/cswatt/advanced-filtering/metrics/summary

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Comment by ijkaylin** [comment]
For the advanced filtering doc: 

- can we bold the "From dropdown" phrase in the first paragraph? 
-  make sure the query strings appear before the photos/ GIFs ?

File: content/en/metrics/advanced-filtering.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+```
+avg:system.cpu.user{env:shop.ist AND availability-zone IN (us-east-1a, us-east-1c, us-east-1d)} by {availability-zone}
+```
+
+
+{{< img src="metrics/advanced-filtering/ex3.gif" alt="Example 3"  style="width:80%;" >}}
+
+```
+avg:system.cpu.user{app NOT IN (village)} by {app}
+```
```
Comment (ruthnaebeck) [inline content/en/metrics/advanced-filtering.md:57]: Add this to the bottom:

```
## Further Reading

{{< partial name="whats-next/whats-next.html" >}}
```

File: content/en/metrics/advanced-filtering.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    tag: "Documentation"
+    text: "Metrics Summary"
+  - link: "/metrics/distributions/"
+    tag: "Documentation"
+    text: "Metrics Distributions"
+---
+
+## Overview
+
+Regardless of whether you’re using Metrics Explorer, monitors, dashboards, or notebooks to query metrics data, you can filter the data to narrow the scope of the timeseries returned. Any metric can be filtered by tag(s) using the from dropdown to the right of the metric.
```
Comment (ruthnaebeck) [inline content/en/metrics/advanced-filtering.md:19]:
```suggestion
Regardless of whether you’re using the Metrics Explorer, monitors, dashboards, or notebooks to query metrics data, you can filter the data to narrow the scope of the timeseries returned. Any metric can be filtered by tag(s) using the from dropdown to the right of the metric. 
```

File: content/en/metrics/summary.md [hunk 3]
```diff
@@ -18,62 +18,83 @@ further_reading:
...
 The [Metrics Summary page][1] displays a list of your metrics reported to Datadog under a specified time frame: the past hour, day, or week. Search your metrics by name or tag using the **Metric** or **Tag** search fields:
 
-{{< img src="metrics/summary/tagexplorer.gif" alt="Filter by Tag"  style="width:75%;">}}
+{{< img src="metrics/summary/tagexplorer2.gif" alt="Filter by Tag"  style="width:75%;">}}
 
-## Metric panel
+## Metric details sidepanel
 
-Click on a metric name to display a panel with more detailed information. The metric panel displays key information for a given metric.
+Click on any metric name to display its details sidepanel for more detailed information regarding the metric’s metadata and tags:
```
Comment (ruthnaebeck) [inline content/en/metrics/summary.md:25]:
```suggestion
Click on any metric name to display its details sidepanel for more information regarding the metric’s metadata and tags: 
```

File: content/en/metrics/summary.md [hunk 4]
```diff
@@ -18,62 +18,83 @@ further_reading:
...
+The name of your metric in the [Metrics Explorer][2], [dashboards][3], etc.
 
-### Number of distinct metrics
+### Distinct Metrics Reported
 
-A metric can correspond to multiple distinct metrics depending on its associated tags. Learn more in the [custom metrics][4] documentation.
+A metric name may emit multiple distinct metrics depending on its associated tag value combinations. This number varies with the timeframe chosen on the page.
 
-### Hosts
+Learn more in the [counting custom metrics][4] documentation.
```
Comment (ruthnaebeck) [inline content/en/metrics/summary.md:37]:
```suggestion
Learn more in the [custom metrics][4] documentation.
```

File: content/en/metrics/summary.md [hunk 5]
```diff
@@ -18,62 +18,83 @@ further_reading:
...
 
-The total number of hosts reporting a metric along with a list of hosts. Click on any host to filter the metrics summary or view the filtered metric in the [metrics explorer][2].
+### Hosts
 
-### Tags
+The total number of hosts reporting a metric .
```
Comment (ruthnaebeck) [inline content/en/metrics/summary.md:41]:
```suggestion
The total number of hosts reporting a metric.
```

File: content/en/metrics/summary.md [hunk 6]
```diff
@@ -18,62 +18,83 @@ further_reading:
...
 
-The total number of tags attached to a metric along with a list of tags. Click on any tag to filter the metrics summary or view the filtered metric in the [metrics explorer][2]:
+### Tag Values
```
Comment (ruthnaebeck) [inline content/en/metrics/summary.md:43]:
```suggestion
### Tag values
```

File: content/en/metrics/summary.md [hunk 7]
```diff
@@ -18,62 +18,83 @@ further_reading:
...
+The metric description helps you understand what a metric does. Descriptions are pre-populated for metrics coming from supported [integrations][9]. Use this field to update the descriptions for your [custom metrics][4].
+
+### Tags table
+
+The tags table offers a variety of ways to explore all the tag keys and tag values that are actively reporting in your metric’s data.
+
+Use the tags table to:
+
+- Sort tag keys by the **Count column** (count of unique tag values).
+- Quickly search through the paginated table of tags for a particular tag key.
```
Comment (ruthnaebeck) [inline content/en/metrics/summary.md:84]:
```suggestion
- Search through the paginated table of tags for a particular tag key.
```

File: content/en/metrics/summary.md [hunk 8]
```diff
@@ -18,62 +18,83 @@ further_reading:
...
+- Export the tags table as a downloadable CSV.
+
+For any particular tag key, you can:
+
+- Inspect all tag values of that tag key.
+- Use a specific tag key:value to further filter the list of metrics displayed on the Metrics Summary page .
```
Comment (ruthnaebeck) [inline content/en/metrics/summary.md:90]:
```suggestion
- Use a specific tag `key:value` to further filter the list of metrics displayed on the Metrics Summary page.
```

File: content/en/metrics/summary.md [hunk 9]
```diff
@@ -18,62 +18,83 @@ further_reading:
...
+- Open a graph of this metric filtered by your tag key:value pair in Metrics Explorer.
```
Comment (ruthnaebeck) [inline content/en/metrics/summary.md:91]:
```suggestion
- Open a graph of this metric filtered by your tag `key:value` pair in the Metrics Explorer.
```

File: content/en/metrics/summary.md [hunk 10]
```diff
@@ -18,62 +18,83 @@ further_reading:
...
+- Copy any tag key:value for easier filtering across the application.
```
Comment (ruthnaebeck) [inline content/en/metrics/summary.md:92]:
```suggestion
- Copy any tag `key:value` for filtering across the application.
```

**Review by ruthnaebeck — APPROVED** [review]
👍

**Comment by ijkaylin** [comment]
Last edits for Advanced Filtering doc: 
Can we change this query to match the one in the GIF? 
_avg:system.cpu.user{env:shop.ist AND availability-zone IN (us-east-1a, us-east-1c, us-east-1d)} by {availability-zone}_ should now be

`avg:system.cpu.user{env:shop.ist AND availability-zone IN (us-east-1a, us-east-1b, us-east-4b)} by {availability-zone}`

** Edits for Metrics Summary ** 
* https://docs-staging.datadoghq.com/cswatt/advanced-filtering/metrics/summary/#metric-type: The type for your metric (gauge, rate, count, distribution). See the metric type page for more details.

Warning: Editing the metric type changes that metric’s behavior for ALL your dashboards and monitors.

**Review by ijkaylin — CHANGES_REQUESTED** [review]

File: content/en/metrics/_index.md [hunk 11]
```diff
@@ -79,67 +90,66 @@ _Aggregation_ defines how the metrics in each group are combined. There are four
...
 
-A **_distribution_** is similar to a histogram, but it summarizes values submitted during a time interval across all hosts in your environment. You can also choose to report multiple percentiles: p50, p75, p90, p95, and p99. You can learn more about this powerful feature [here][18].
+A **_distribution_** is similar to a histogram, but it summarizes values submitted during a time interval across all hosts in your environment. You can also choose to report multiple percentiles: p50, p75, p90, p95, and p99. You can learn more about this powerful feature in the [Distributions documentation][18].
 
 To make this concrete with an example, suppose your host reported metric values of [1,1,1,2,2,2,3,3] during a ten-second interval. Depending on the metric type you chose, Datadog would store completely different values:
```
Comment (ijkaylin) [inline content/en/metrics/_index.md:133]:
```suggestion
```

File: content/en/metrics/_index.md [hunk 12]
```diff
@@ -79,67 +90,66 @@ _Aggregation_ defines how the metrics in each group are combined. There are four
...
 
-_Count_ would add them up and send the value 15 over to our servers, while _rate_ would take the total sum and divide it by 10 seconds to report a value of 1.5. _Gauge_ would simply report the last value, 3. If your metric is a _histogram_, Datadog would receive five different values: avg = 1.88, count = 8, median = 2, p95 = 3, max = 3.
+_Count_ would add them up and send the value 15 over to Datadog's servers, while _rate_ would take the total sum and divide it by 10 seconds to report a value of 1.5. _Gauge_ would simply report the last value, 3. If your metric is a _histogram_, Datadog would receive five different values: avg = 1.88, count = 8, median = 2, p95 = 3, max = 3.
```
Comment (ijkaylin) [inline content/en/metrics/_index.md:135]:
```suggestion
```

File: content/en/metrics/_index.md [hunk 13]
```diff
@@ -79,67 +90,66 @@ _Aggregation_ defines how the metrics in each group are combined. There are four
...
 
-Metric types also determine which graphs and functions are available to use with the metric in the app. Please see the [metrics types][15] documentation for more detailed examples of each metric type and submission instructions.
+Metric types also determine which graphs and functions are available to use with the metric in the app. See the [metrics types][15] documentation for more detailed examples of each metric type and submission instructions.
```
Comment (ijkaylin) [inline content/en/metrics/_index.md:137]:
```suggestion
See the [metrics types][15] documentation for more detailed examples of each metric type and submission instructions.
```

**Review by ijkaylin — CHANGES_REQUESTED** [review]

**Review by apigirl — APPROVED** [review]

**Pull request merged**